rosidl_generator_c/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ add_library(${PROJECT_NAME} SHARED
 )
 if(NOT WIN32)
   set_target_properties(${PROJECT_NAME} PROPERTIES
-    COMPILE_FLAGS "-std=c11 -Wall -Wextra")
+    COMPILE_FLAGS "-Wall -Wextra")
```

Comment:
We don't use C11 anywhere. The only thing we use is `stdatomics.h` in `rcl`, but `-std=c11` doesn't affect it at all.